demo/GraphTutorial/app/src/main/java/com/example/graphtutorial/AuthenticationHelper.java

```diff
@@ -40,11 +47,25 @@ public void onError(MsalException exception) {
                 });
     }
 
-    public static synchronized void getInstance(Context ctx, IAuthenticationHelperCreatedListener listener) {
+    public static synchronized CompletableFuture<AuthenticationHelper> getInstance(Context ctx) {
```

Comment:
we probably want to avoid showing static things when possible. It might work in Android, but should anyone come from a server app, they'll run into thread safety issues.